jiant/utils/tokenization_normalization.py

```diff
@@ -67,6 +67,13 @@ def normalize_tokenizations(
         modifed_space_tokenization = bow_tag_tokens(space_tokenization)
         modifed_target_tokenization = _process_sentencepiece_tokens(target_tokenization)
     else:
+        import sys
+
+        if "pytest" in sys.modules:
+            from jiant.utils.testing.tokenizer import SimpleSpaceTokenizer
+
+            if isinstance(tokenizer, SimpleSpaceTokenizer):
+                return space_tokenization, target_tokenization
```

Comment:
`SimpleSpaceTokenizer` is used for tests, however it is not type `transformers.PretrainedTokenizer` as expected by this function.

Reply:
Mind pulling out the `pytest` check to be its own function? We can put it in `utils.testing`